functest/deploy_test.go

```diff
@@ -172,14 +172,13 @@ func TestDeployNamespaceMismatch(t *testing.T) {
 	t.Logf("Running appsody deploy with namespace: %s ...", secondNamespace)
 	output, err := cmdtest.RunAppsody(sandbox, "deploy", "--generate-only", "-n", secondNamespace)
 
+	expectedOutput := "Overriding namespace " + firstNamespace + " in the deployment manifest to: " + secondNamespace
+	if !strings.Contains(output, expectedOutput) {
+		t.Errorf("Did not get expected output: %s", expectedOutput)
+	}
```

Comment:
Just a suggestion, we could check that the deployment manifest contains the correct namespace